src/nxt_sockaddr.c

```diff
@@ -572,8 +573,8 @@ static nxt_sockaddr_t *
 nxt_sockaddr_unix_parse(nxt_mp_t *mp, nxt_str_t *addr)
 {
 #if (NXT_HAVE_UNIX_DOMAIN)
-    size_t          length, socklen;
-    u_char          *path;
+    size_t          socklen;
+    nxt_str_t       *path;
```

Comment:
Hi Alex,

What's the motivation for switching path to a nxt_str_t?

I'm not seeing an obvious reason and not doing so would keep the patch simpler...

Hmm, maybe I'm missing something, but that seems to be the only change here... or maybe I should look at this thing properly in git!

Reply:
You're right.  I did a different change which needed the nxt_str_t, and then I realized I could simplify a lot of it, but didn't see that I could drop even more changes.  :)